src/components/Icons.tsx

```
@@ -0,0 +1,206 @@
+import type React from "react"
+import { View, Image, Text } from "react-native"
```

ESLint

[error] 'Text' import from 'react-native' is restricted. Use <AppText> instead of <Text>. The <AppText> component is located in 'components/AppText'. We are using this because it has a custom font and some other features.

(no-restricted-imports)